CircleandCross2/Program.cs

```diff
@@ -20,32 +20,86 @@ static void GameBoard (string[,] board)
 			
 		}
 
+		static bool GameWonCondition(string[,] board)
+		{
+			for (int i = 0; i < 3; i++)
```

Comment:
Why do you use nested for here?

Reply:
To create board

Reply:
The board already exists. It is passed as an argument to this function.
You need to have two for loops, but not nested ones.


```csharp
static bool CheckWinCondition(string[,] board, string playersToken)
{
    for (int rows = 0; rows < 3; rows++)
    {
        //Check rows here
    }

    for (int columns = 0; columns < 3; columns++)
    {
        //check columns here
    }

    if (/*Check diagonals here*/)
    {
        return true;
    }

    return false;
}

```

Reply:
Now only diagonals work, i dont use srting playertoken becouse i dont now how use him

Reply:
you can pass an argument which represents player:
```csharp
CheckWinCondition(array, "[x]");
CheckWinCondition(array, "[o]");
```

Reply:
i have this but i still use only string[,] board

Reply:
I do it

Reply:
i do did